proposal/comp110_architecture_proposal.tex

```diff
@@ -17,58 +17,58 @@ \section*{Topic}
 \end{description}
 
 \section*{Paper 4}
 \begin{description}
-\item[Title:] Title of paper
-\item[Citation:] \cite{bibtex_key}
-\item[Abstract:] Copy and paste the abstract here
-\item[Web link:] Give the URL of the paper in IEEE Xplore, ACM Digital Library, or similar
-\item[Full text link:] Give the URL of a downloadable PDF of the paper, if you can find one
-\item[Comments:] Write a few sentences on how you found the article and why you believe it is relevant and/or important.
+\item[Title:] Polymorph: A Model for Dynamic Level Generation
+\item[Citation:] \cite{Jennings:2010}
+\item[Abstract:] "Players begin games at different skill levels and develop their skill at different rates—so that even the best-designed games are uninterestingly easy for some players and frustratingly difficult for others. A proposed answer to this challenge is Dynamic Difficulty Adjustment (DDA), a general category of approaches that alter games during play, in response to player performance. However, nearly all these techniques are focused on basic parameter tweaking,
+while the difficulty of many games is connected to aspects that are more challenging to adjust dynamically, such as level design. Further, most DDA techniques are based on designer intuition, which may not reflect actual play patterns. Responding to these challenges, we have created Polymorph, which employs techniques from level generation and machine learning to understand level difficulty and player skill, dynamically constructing levels for a 2D platformer game with continually-appropriate challenge. We present the results of the user study on which Polymorph's model of level difficulty is based, as well as a discussion of the unique features of the model. We believe Polymorph creates a play experience that is unique because the changes are both personalized and structural, while also providing an example of a new application of machine learning to aid game design.
+\item[Web link:] \url{http://aaai.org/ocs/index.php/AIIDE/AIIDE10/paper/view/2150}
+\item[Full text link:] \url{https://www.aaai.org/ocs/index.php/AIIDE/AIIDE10/paper/.../2558}
+\item[Comments:] This article suggests that the game level should be generatored with reference from the player's skill level and increased learning rate. It suggests a way to adjust the game level during play.
 \end{description}
 
 \section*{Paper 5}
 \begin{description}
-\item[Title:] Title of paper
-\item[Citation:] \cite{bibtex_key}
-\item[Abstract:] Copy and paste the abstract here
-\item[Web link:] Give the URL of the paper in IEEE Xplore, ACM Digital Library, or similar
-\item[Full text link:] Give the URL of a downloadable PDF of the paper, if you can find one
-\item[Comments:] Write a few sentences on how you found the article and why you believe it is relevant and/or important.
+\item[Title:] {Patterns and procedural content generation: revisiting Mario in world 1 level 1}
```

Comment:
Another interesting paper, however maybe a little light on technical detail and empirical results for this assignment. Can you find any other papers by Dahlskog and Togelius which go into more detail?